relay/instance.go

```diff
@@ -0,0 +1,32 @@
+package relay
+
+import (
+	"net"
+
+	"github.com/Layr-Labs/eigensdk-go/logging"
+)
+
+// Instance holds the state for a single relay server instance.
+type Instance struct {
+	Server   *Server
+	Listener net.Listener
+	Port     string
+	URL      string
+	Logger   logging.Logger
+}
```

Comment:
why do we need this? There's too many structs flowing around. It should be simple:
1. create config
2. create relay struct
3. start()

The "server" here in instance is not a "server", its like... the whole relay thing.
Is there a way to start/use the relay without the grpc server? If not, then I'd say just stick the server in the Relay (Server) struct itself.

Reply:
https://github.com/Layr-Labs/eigenda/pull/2107/commits/a8c93ca0c24cfb0d2bea58b847e45a20ab69c91c